contrib/test/ci/vars.yml

```diff
@@ -169,6 +169,7 @@ kata_skip_network_tests:
   - 'test "Clean up network if pod sandbox gets killed"'
 kata_skip_pod_tests:
   - 'test "pass pod sysctls to runtime"'
+  - 'test "pass pod sysctls to runtime when in userns"'
```

Comment:
Should we create an issue for this to track the failure?

Reply:
we could though hostUsers isn't super meaningful in a kata context because the IDs are already separated from the host through virtualization